### PR TITLE
fix(app): replace graphql-upload to graphql-upload-minimal #18 #16

### LINKED ISSUE #16: ESM support.
Is there a plan to introduce ESM (ECMAScript Module) support? The new version 16 of `graphql-upload`, which is a dependent is already in ESM and therefore not working with this version.

### LINKED ISSUE #18: On Image exceed file size, server stop !
Hello,
I'm trying to use mercurius upload but I have an error who stop my server if the uploaded file exceed size.

```bash 
[15:48:28.141] DEBUG (152219): File truncated as it exceeds the 131100 byte size limit.
    err: {
      "type": "PayloadTooLargeError",
      "message": "File truncated as it exceeds the 131100 byte size limit.",
      "stack":
          PayloadTooLargeError: File truncated as it exceeds the 131100 byte size limit.
              at FileStream.<anonymous> (/app/node_modules/graphql-upload/processRequest.js:250:23)
              at FileStream.emit (node:events:513:28)
              at SBMH.ssCb [as _cb] (/app/node_modules/busboy/lib/types/multipart.js:479:32)
              at feed (/app/node_modules/streamsearch/lib/sbmh.js:248:10)
              at SBMH.push (/app/node_modules/streamsearch/lib/sbmh.js:104:16)
              at Multipart._write (/app/node_modules/busboy/lib/types/multipart.js:567:19)
              at writeOrBuffer (node:internal/streams/writable:392:12)
              at _write (node:internal/streams/writable:333:10)
              at Multipart.Writable.write (node:internal/streams/writable:337:10)
              at IncomingMessage.ondata (node:internal/streams/readable:766:22)
      "status": 413,
      "statusCode": 413,
      "expose": true
    }
{
  filename: 'corey-willett-4wuVqpRvqbY-unsplash.jpg',
  mimetype: 'image/jpeg',
  encoding: '7bit',
  createReadStream: [Function: createReadStream]
}
[15:48:28.142] DEBUG (152219): Server is shutting down
```

```ts
@Mutation(() => String)
  async updateProfilePicture(
    @Arg('image', () => GraphQLUpload, { nullable: true }) payload,
  ): Promise<string> {
    await Controller.updateProfilePicture(payload);

    return 'OK';
  }
  
  
class Controller {
  static async updateProfilePicture(userId: string, payload) {
    //Retrict uploaded file
    if (['image/jpeg', 'image/png', 'image/jpg'].indexOf(payload.mimetype) === -1) {
      throw new BadRequest({ error: GenericErrors.invalid_file_type });
    }

    try {
      const rs = payload.createReadStream(); <------ C R A S H    H E R E
    } catch (e) {
      console.log(e);
      throw new BadRequest({ error: GenericErrors.file_too_large });
    }

    console.log(payload);
    // await savim.uploadFile(`nanny/${}`)
  }
}

export default Controller;

### PR DESCRIPTION
**Pull request by qlaffont**
Fixes #16
Fixes #18

### CODE REVIEW

**Comment by qlaffont** [comment]
@mcollina @PabloSzx :wave:

**Comment by qlaffont** [comment]
4 days since PR, If in 1 week no review and merge I will do a fork :/

**Comment by mcollina** [comment]
(I've been on vacation for the last two weeks)

I don't understand why this PR fixes the two linked issues.

What are the difference with graphql-upload-minimal?

**Comment by qlaffont** [comment]
My bad didn't know ! 

- Bug for fix
Mercurius was crashing according how graphql-upload handle upload

- ESM
This fork of graphql-upload have better performance and handle both case (CJS/ESM)

File: tsconfig.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     "skipLibCheck": true /* Skip type checking of declaration files. */,
     "forceConsistentCasingInFileNames": true, /* Disallow inconsistently-cased references to the same file. */
 
-    /* https://github.com/jaydenseric/graphql-upload/issues/282#issuecomment-1142691771 */
+    /* https://github.com/flash-oss/graphql-upload-minimal/issues/282#issuecomment-1142691771 */
```
Comment (mcollina) [inline tsconfig.json:69]: This seems wrongly renamed, could you check?

**Review by mcollina — APPROVED** [review]
lgtm with that tiny nit

File: tsconfig.json (hunk 1 above)
Comment (qlaffont) [reply tsconfig.json:69]: yes I remove the comment is useless in this case

**Pull request merged**